src/app/api/chat/route.ts

```
@@ -0,0 +1,304 @@
+    model: openrouter.chat("openrouter/auto"),
+  });
+
+  // Use toTextStreamResponse or toUIMessageStreamResponse based on what is available in the installed SDK version
+  if (typeof (result as any).toDataStreamResponse === 'function') {
+    return (result as any).toDataStreamResponse();
+  } else if (typeof (result as any).toUIMessageStreamResponse === 'function') {
+    return (result as any).toUIMessageStreamResponse();
+  } else {
+    return result.toTextStreamResponse();
+  }
+  } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Replace `'API·ERROR:·'·+·err.message·+·'\n'·+·err.stack,·{·status:·500` with `"API·ERROR:·"·+·err.message·+·"\n"·+·err.stack,·{⏎······status:·500,⏎···`

(prettier/prettier)